net/DevExtreme.AspNet.Data/DataSourceLoadOptionsBase.cs

```diff
@@ -67,12 +67,21 @@ public class DataSourceLoadOptionsBase {
         /// </summary>
         public string[] Select { get; set; }
 
+        /// <summary>
+        /// An array of data fields that constrains the <see cref="Select" /> expression.
```

Comment:
constrains? Explain.

Reply:
Say there are 100 fields in the data source. Without `PreSelect`, it's possible to use any of those 100 in `Select`.
With `PreSelect`, the developer fixes/constrains fields that are allowed for usage in `Select`. Not allowed fields specified in `Select` are ignored without errors.